.github/CONTRIBUTING.md

```diff
@@ -2,60 +2,110 @@
 
 Contributor guidelines for the Location Open Source Tracker project.
 
+First off, thanks for taking the time to contribute!
+
+## Aim
+This project seeks to provide an open source alternative to Google's [Fused Location Provider](https://developer.android.com/training/location/index.html), only depending on the Android SDK.
+Therefore, contributions should implement the [FusedLocationProviderAPI](https://developers.google.com/android/reference/com/google/android/gms/location/FusedLocationProviderApi) or fix bugs/improve the current implementation, rather than implementing completely new features.
```

Comment:
I would add here that the [GeoencingApi](https://developers.google.com/android/reference/com/google/android/gms/location/GeofencingApi) and [SettingsApi](https://developers.google.com/android/reference/com/google/android/gms/location/SettingsApi) are also supported in addition to `FusedLocationProviderApi`. 

Basically our goal is to support an open source implementation of anything in the the [com.google.android.gms.location](https://developers.google.com/android/reference/com/google/android/gms/location/package-summary) package.